azure_functions_worker/dispatcher.py

```diff
@@ -261,6 +263,7 @@ async def _handle__worker_init_request(self, req):
             constants.TYPED_DATA_COLLECTION: _TRUE,
             constants.RPC_HTTP_BODY_ONLY: _TRUE,
             constants.RPC_HTTP_TRIGGER_METADATA_REMOVED: _TRUE,
+            constants.SHARED_MEMORY_DATA_TRANSFER: _TRUE,
```

Comment:
You may probably want to wrap this capability under a feature flag, e.g.
```python
if is_envvar_true(constants.PYTHON_USE_SHARED_MEMORY_DATA_TRANSFER):
    constants.SHARED_MEMORY_DATA_TRANSFER: _TRUE
```

Reply:
Do you suggest having another flag called `PYTHON_USE_SHARED_MEMORY_DATA_TRANSFER` in addition to `SHARED_MEMORY_DATA_TRANSFER`? So per-language flag?

Reply:
@gohar94, good catch!
This is not a language specific flag, so please use `FUNCTIONS_WORKER_SHARED_MEMORY_DATA_TRANSFER_ENABLED` instead.